index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books ES6</title>
+    <link rel="stylesheet" href="index.css">
+    <script type="module" src="index.js" defer></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header">
+        <section class="navigations">    
+            <h1 class="web-page">A Web Page</h1>
+            <div class="navigation-contents">
+                <i class="fa-solid fa-left-long fa-3x white"></i>
+                <i class="fa-solid fa-right-long white fa-3x"></i>
+                <i class="fa-solid fa-xmark fa-3x"></i>
+                <i class="fa-solid fa-house fa-2x white"></i>
+                <input type="text" placeholder="https://" class="browser-bar">
+                <div class="search-bar">
+                  <input type='text' placeholder='Search...' class='search-input'/><i class="fa-solid fa-magnifying-glass"></i>
+                </div>
+            </div>
+           
+        </section>
```

Comment:
- [ ] Nice work so far 👍🏿  but this section of the project is unnecessary. Please i advice you remove it 😄